ostelco-ios-client/Coordinators/OnboardingCoordinator.swift

```diff
@@ -476,8 +476,6 @@ extension OnboardingCoordinator: ESIMInstructionsDelegate {
         primeAPI.loadContext()
         .then { (context) -> PromiseKit.Promise<PrimeGQL.SimProfileFields> in
             assert(context.regions.count == 1)
-            // swiftlint:disable:next empty_count
-            assert(context.regions.first!.fragments.regionDetailsFragment.simProfiles?.count == 0)
             let simProfile = RegionResponse.getRegionFromRegionResponseArray(context.regions.map({ $0.fragments.regionDetailsFragment }))?.getSimProfile()
```

Comment:
@havard024 See my comment in the patch for this.

Reply:
I agree that the assert does not make sense according how we want this to work.